analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp7.cs

```diff
@@ -108,11 +108,14 @@ void Patterns_In_Loops(object o, object[] items, int length)
         void Patterns_In_Loops_With_Discard(object o, object[] items)
         {
             // The following should not throw exceptions
-            while (o is string _) { }
+            while (o is string _)
+            { }
 
-            do { } while (o is string _);
+            do
+            { } while (o is string _);
 
-            for (int i = 0; i < items.Length && items[i] is string _; i++) { }
+            for (int i = 0; i < items.Length && items[i] is string _; i++)
+            { }
```

Comment:
Formatting was broken in this file. I did "format document". Turn "Hide whitespace" on in GitHub to have a better review experience in this document.

Reply:
..but thanks for the hint, that was useful